fullstop/src/main/resources/db/migration/V6__create_outdated_ami_type.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO fullstop_data.violation_type (
```

Comment:
About the move of the db folder. Is a good idea that both application can change the schema? Example: fullstop is running and fullstop job change the schema, not sure if is fine.